content/actions/using-workflows/reusing-workflows.md

```diff
@@ -103,7 +103,11 @@ You can define inputs and secrets, which can be passed from the caller workflow
            required: true
    ```
    {% endraw %}
+   {% if actions-inherit-secrets-reusable-workflows %}
+   For details of the syntax for defining inputs and secrets, see [`on.workflow_call.inputs`](/actions/reference/workflow-syntax-for-github-actions#onworkflow_callinputs), [`on.workflow_call.secrets`](/actions/reference/workflow-syntax-for-github-actions#onworkflow_callsecrets) and [`on.workflow_call.secrets.inherit`](/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callsecretsinherit).
+   {%else%}
    For details of the syntax for defining inputs and secrets, see [`on.workflow_call.inputs`](/actions/reference/workflow-syntax-for-github-actions#onworkflow_callinputs) and [`on.workflow_call.secrets`](/actions/reference/workflow-syntax-for-github-actions#onworkflow_callsecrets).
+   {%endif%}
 1. In the reusable workflow, reference the input or secret that you defined in the `on` key in the previous step.
```

Comment:
Here we might want to mention that the undefined secrets can be referenced if we use `inherit`?
```suggestion
1. In the reusable workflow, reference the input or secret that you defined in the `on` key in the previous step. If the secrets are inherited using `secrets: inherit`, secrets that are not defined in the `on` key can be referenced.
```